packages/docusaurus-theme-classic/codeTranslations/ja.json

```diff
@@ -43,13 +43,13 @@
   "theme.docs.sidebar.responsiveCloseButtonLabel": "メニューを閉じる",
   "theme.docs.sidebar.responsiveOpenButtonLabel": "メニューを開く",
   "theme.docs.versions.latestVersionLinkLabel": "最新バージョン",
-  "theme.docs.versions.latestVersionSuggestionLabel": "最新のドキュメントは{latestVersionLink} ({versionLabel}) を見てください。",
+  "theme.docs.versions.latestVersionSuggestionLabel": "最新のドキュメントは{latestVersionLink} ({versionLabel})を見てください。",
```

Comment:
The space after the parentheses is required in Japanese as well.

Reply:
It's true :). Actually, I was originally adding a space between English letters and Japanese ones everywhere, which is a common typesetting style (the [react website](https://ja.reactjs.org/) in Japanese follows this style for example), but since there's no standard rule for this, and the "no space between half-width and full-width characters (i.e. EN letters or half-width numbers/JP characters)" style is also very common in Japanese (I think we both agree on this), I ended up adopting the latter style and not adding spaces (and thus removed this space, which turns out to be unnecessary).

Reply:
Those labels are "default labels" that you can always override anyway

What does make sense for the majority of cases?